index.src.html

```diff
@@ -1477,41 +1481,24 @@
         adjustments.
   </dd>
 
-  <dt><dfn dict-member for=VideoDecoderConfig>cropLeft</dfn></dt>
-  <dd>
-    The number of pixels to remove from the left of the VideoFrame, prior to
-        aspect ratio adjustments. Defaults to zero if not present.
-  </dd>
-
-  <dt><dfn dict-member for=VideoDecoderConfig>cropTop</dfn></dt>
-  <dd>
-    The number of pixels to remove from the top of the VideoFrame, prior to
-        aspect ratio adjustments. Defaults to zero if not present.
-  </dd>
+  NOTE: {{VideoDecoderConfig/codedWidth}} and {{VideoDecoderConfig/codedHeight}}
+    are only used as hints when selecting a {{VideoDecoder/[[codec implementation]]}}.
 
-  <dt><dfn dict-member for=VideoDecoderConfig>cropWidth</dfn></dt>
+  <dt><dfn dict-member for=VideoDecoderConfig>displayAspectWidth</dfn></dt>
   <dd>
-    The width in pixels to include in the crop, starting from cropLeft.
-        Defaults to codedWidth if not present.
+    Horizontal component of the VideoFrame's aspect ratio when displayed.
   </dd>
 
-  <dt><dfn dict-member for=VideoDecoderConfig>cropHeight</dfn></dt>
+  <dt><dfn dict-member for=VideoDecoderConfig>displayAspectHeight</dfn></dt>
   <dd>
-    The height in pixels to include in the crop, starting from cropLeft.
-        Defaults to codedHeight if not present.
+    Vertical component of the VideoFrame's aspect ratio when displayed.
   </dd>
 
-  <dt><dfn dict-member for=VideoDecoderConfig>displayWidth</dfn></dt>
-  <dd>
-    Width of the VideoFrame when displayed. Defaults to cropWidth if not
-        present.
-  </dd>
-
-  <dt><dfn dict-member for=VideoDecoderConfig>displayHeight</dfn></dt>
-  <dd>
-    Height of the VideoFrame when displayed. Defaults to cropHeight if not
-        present.
-  </dd>
+Note: {{VideoFrame/displayWidth}} and {{VideoFrame/displayHeight}} can both be
+  different from {{VideoDecoderConfig/displayAspectWidth}} and
+  {{VideoDecoderConfig/displayAspectHeight}}, but they should have identical
+  ratios, after being scaled according to defintions of
+  [=intrinsic width|intrinsic size=].
```

Comment:
I'd replace this link w/ a link to the create videoframe algo.

Reply:
Done.